pages/1.13/security/oss/managing-users/external-users/index.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: layout.pug
+navigationTitle: External users
+title: External users
+excerpt: Managing external DC/OS users
+menuWeight: 10
+---
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+An external user is automatically created for the first user who logs in to the DC/OS cluster via a single sign-on flow through Auth0.
+
+# Log in external users via the web interface
+
+1.  Launch the DC/OS web interface.
+
+2.  Log in through a single sign-on flow using the identity provider of your choice (Google, GitHub, and Microsoft).
+
+**NOTE**: Additional users must be added by another authorized user before they can log in to DC/OS.
+
+
+# Log in external users via the DC/OS CLI
+
+1.  To log in to the DC/OS CLI, enter the [auth login](/1.13/cli/command-reference/dcos-auth/dcos-auth-login/) command:
+
+```bash
+dcos auth login --provider dcos-oidc-auth0
+If your browser didn't open, please go to the following link:
+
+    http://172.17.0.2/login?redirect_uri=urn:ietf:wg:oauth:2.0:oob
+
+Enter OpenID Connect ID Token: 
+>
+```
+
+2. Follow the displayed instructions to trigger the browser login flow.
+
+3. Copy the `OpenID Connect ID token` from the modal dialog that appears in the browser.
+
+4. Paste the `OpenID Connect ID token` into the DC/OS CLI for login completion.
+
+**NOTE**: The `--provider` argument is set to `dcos-oidc-auth0` by default.
+
+# Add external users from the web interface
+
+1.  Log in to the web interface with your username (Google, GitHub, and Microsoft) and password.
+
+2.  Click on  **Organization** in the left hand menu. From the **Users** screen, click the plus sign (**+**) in the upper right corner, and fill in the new user email address. New users are automatically sent an email notifying them of access to DC/OS.
```

Comment:
Does the email work in a default setup? Do we need to link to instructions on how to enable email notification?

Reply:
Hmm I tried it and it seems to not work out of the box on my `minidcos` cluster.
@jgehrcke do you have any insight on how to enable this feature?

Reply:
> New users are automatically sent an email notifying them of access to DC/OS.

let's remove this from the docs